include/worker.hpp

```diff
@@ -100,6 +100,16 @@ class Worker
                       types::ObjectMap& objectInterfaceMap,
                       const std::string& vpdFilePath);
 
+    /**
```

Comment:
Could you please remind me why this was needed ?

Reply:
we decided not to create n copies of json object. so always get from worker class.